app/views/dashboard/accounts/settings.html.erb

```diff
@@ -15,7 +15,10 @@
         <p class="mb-4"><%= current_user.email %></p>
 
         <%= link_to "Change Email or Password", edit_user_registration_path %> <br>
-        <%= link_to "Log out", destroy_user_session_path, method: :delete %>
+        <%= button_to "Log out", destroy_user_session_path,
+            method: :delete,
+            class: "no-underline cursor-pointer text-mapzy-red hover:text-mapzy-red-light font-semibold"
+        %>
```

Comment:
Weird that it doesn't work with `link_to` and `method: :delete`, even though it's written in the official [documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to). I also kind of remember trying this out and it worked. Maybe a browser thing?

Reply:
thanks anyways for fixing it

Reply:
It's a Rails 7 + Turbo thing. Turbo is annoying the shit out of me, haha.